include/kllvm/bindings/core/core.h

```diff
@@ -37,6 +37,10 @@ std::shared_ptr<kllvm::KOREPattern> simplify(
 bool is_sort_kitem(std::shared_ptr<kllvm::KORESort> const &sort);
 bool is_sort_k(std::shared_ptr<kllvm::KORESort> const &sort);
 
+std::shared_ptr<kllvm::KOREPattern> evaluate_function(
+    std::string const &label,
+    std::vector<std::shared_ptr<kllvm::KOREPattern>> const &args);
```

Comment:
Did you consider passing a `CompositePattern`?

Reply:
Right, since part of the input is already structured (`args`), I guess having a single structured parameter would make this a bit cleaner.